featuretools/computational_backends/pandas_backend.py

```diff
@@ -151,15 +170,48 @@ def calculate_all_features(self, instance_ids, time_last,
                 if not self.entityset.find_backward_path(start_entity_id=filter_eid,
                                                          goal_entity_id=eid):
                     entity_frames[eid] = eframes_by_filter[eid][eid]
+                    # precalculated features will only be placed in entity_frames,
```

Comment:
is it possible to have to have precalculated a `needs_all_values` feature? do we need to add code to handle that case?

Reply:
i thought that I considered all cases before but I did miss the case when you have a `needs_all_values` feature that depends on an approximated/precalculated feature. This will need to be reworked.